src/common/components/userMatchModal/userMatchModal.component.spec.js

```diff
@@ -25,7 +25,7 @@ describe( 'userMatchModal', function () {
 
   describe( '$onInit', () => {
     beforeEach( () => {
-      spyOn( $ctrl.verificationService, 'getContacts' ).and.returnValue( Observable.of( [] ) );
+      spyOn( $ctrl.verificationService, 'getContacts' ).and.callFake( () => Observable.of( [] ) );
```

Comment:
I know why I changed this. I was testing out passing back a variable, then changing it slightly in each it block. Didn't work though.